### PR TITLE
[easy][dynamo] Remove try except from call_getattr

### PR DESCRIPTION
**Pull request by anijain2305**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #128148
* #129285
* __->__ #129217
* #129015
* #129098

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

File: torch/_dynamo/variables/builtin.py [hunk 1]
```diff
@@ -1546,11 +1546,8 @@ def call_getattr(
                         f"pending mutation on nn module, so graph breaking at {name!r} call"
                     )
 
-            try:
-                # re-read a pending side effect?
-                return tx.output.side_effects.load_attr(obj, name)
-            except KeyError:
-                pass
+        is tx.output.side_effects.has_pending_mutation_of_attr(obj, name)
```
Comment (lezcano) [inline torch/_dynamo/variables/builtin.py:1549]: if

**Review by lezcano — APPROVED** [review]

File: torch/_dynamo/variables/builtin.py (hunk 1 above)
Comment (anijain2305) [reply torch/_dynamo/variables/builtin.py]: Haha, I forgot to rerun ghstack. I was surprised to see the number of faiures.

**Pull request closed without merging**